README.md

```diff
@@ -69,63 +70,84 @@ module.exports = [
+| [consistent-output](docs/rules/consistent-output.md)                     | enforce consistent use of `output` assertions in rule tests                  | ✅ ![badge-recommended-type-checked][] |    |    |    |
+| [no-identical-tests](docs/rules/no-identical-tests.md)                   | disallow identical tests                                                     | ✅ ![badge-recommended-type-checked][] | 🔧 |    |    |
+| [no-only-tests](docs/rules/no-only-tests.md)                             | disallow the test case property `only`                                       | ✅ ![badge-recommended-type-checked][] |    | 💡 |    |
+| [prefer-output-null](docs/rules/prefer-output-null.md)                   | disallow invalid RuleTester test cases where the `output` matches the `code` | ✅ ![badge-recommended-type-checked][] | 🔧 |    |    |
+| [test-case-property-ordering](docs/rules/test-case-property-ordering.md) | require the properties of a test case to be placed in a consistent order     |                                       | 🔧 |    |    |
+| [test-case-shorthand-strings](docs/rules/test-case-shorthand-strings.md) | enforce consistent usage of shorthand strings for test cases with no options |                                       | 🔧 |    |    |
 
 <!-- end auto-generated rules list -->
 
 ## <a name='Presets'></a>Presets
 
-|   | Name                | Description                                                               |
-|:--|:--------------------|:--------------------------------------------------------------------------|
-| ✅ | `recommended`       | enables all recommended rules in this plugin                              |
-|   | `rules-recommended` | enables all recommended rules that are aimed at linting ESLint rule files |
-|   | `tests-recommended` | enables all recommended rules that are aimed at linting ESLint test files |
-|   | `all`               | enables all rules in this plugin                                          |
-|   | `rules`             | enables all rules that are aimed at linting ESLint rule files             |
-|   | `tests`             | enables all rules that are aimed at linting ESLint test files             |
+|         | Name                                             | Description                                                                                                                                                            |
+| :-- | :------------------------- | :--------------------------------------------------------------------------------------- |
+| ✅    | `recommended`                            | enables all recommended rules in this plugin, excluding those requiring type information |
+|         | `recommended-type-checked` | enables all recommended rules in this plugin, including those requiring type information |
```

Comment:
[Question] I don't _love_ that this doesn't have its own emoji 😞 is there some standard practice that can be done?

Reply:
Yes I would just give it a different color checkmark like [☑️](https://emojipedia.org/check-box-with-check) in `.eslint-doc-generatorrc.js`.